rclcpp/wait_set/README.md

```diff
@@ -0,0 +1,23 @@
+# Minimal rclcpp wait-set cookbook recipes
+
+This package contains a few different strategies for creating nodes which use `rclcpp::waitset`
+to wait and handle ROS entities, that is, subscribers, timers, clients, services, guard
+conditions and waitables.
+
+
+* `wait_set.cpp`: Simple example showing how to use the default wait-set with a dynamic
+  storage policy and a sequential (no thread-safe) synchronization policy.
+* `static_wait_set.cpp`: Simple example showing how to use the static wait-set with a static
+  storage policy.
+* `thread_safe_wait_set.cpp`: Simple example showing how to use the thread-safe wait-set with a
+  thread-safe synchronization policy.
+* `wait_set_topics_and_timer.cpp`: Simple example using multiple subscriptions,
+  publishers, and a timer.
+* `wait_set_random_order.cpp`: An example showing user-defined
+  data handling and a random publisher. `executor_random_order.cpp` run the same node logic
+  using `SingleThreadedExecutor` to compare the data handling order.
```

Comment:
This pair of executables looks more like a demo or maybe a tutorial. Remember the purpose of examples are to be copy-paste starting points, not to show that a feature works.

I think we already talked about this, and we agreed to take them here and move them later? If that's the case, then nothing to do here, but I wanted to point it out again.

Reply:
I agree. I think this one should go on demos or tutorials. Is this the only executable we should move?